src/AlpineSkiHouse.Web/Program.cs

```diff
@@ -12,6 +12,7 @@ public class Program
         public static void Main(string[] args)
         {
             var host = new WebHostBuilder()
+                .UseUrls("http://*:5000")
```

Comment:
Is this something we want to have happen for all deployments? Are there any security implications to this?